app/build.gradle.kts

```diff
@@ -91,6 +91,9 @@ android {
             )
             if (isReleaseSigningConfigured) {
                 signingConfig = signingConfigs.getByName("release")
+            } else {
```

Comment:
I have a concern about this, as it might cause someone to accidentally sign a debug build with a release signing key.

On CI, we currently do this:
```
      # A fake signing key to satisfy creating a "release" build
      - name: Export Signing Key
        env:
          SIGNING_KEY_PATH: ${{ format('{0}/release.jks', env.home) }}
        shell: bash
        run: |
          keytool -genkey -v -keystore $SIGNING_KEY_PATH -keypass android -storepass android -alias androiddebugkey -keyalg RSA -keysize 2048 -validity 100000 -dname "CN=, OU=, O=Test, L=, S=, C=" -noprompt
      - name: Build
        timeout-minutes: 20
        env:
          ORG_GRADLE_PROJECT_ZCASH_RELEASE_KEYSTORE_PATH: ${{ format('{0}/release.jks', env.home) }}
          ORG_GRADLE_PROJECT_ZCASH_RELEASE_KEYSTORE_PASSWORD: android
          ORG_GRADLE_PROJECT_ZCASH_RELEASE_KEY_ALIAS: androiddebugkey
          ORG_GRADLE_PROJECT_ZCASH_RELEASE_KEY_ALIAS_PASSWORD: android
        run: |
          ./gradlew :app:assembleDebug :app:bundleRelease :app:packageZcashmainnetReleaseUniversalApk
```

However that is quite verbose and doesn't help for local development.  What if we eliminate all of that and just add a new property `IS_SIGN_RELEASE_BUILD_WITH_DEBUG_KEY=false` in the gradle.properties? 

Then on CI or locally someone could add the property like this: `./gradlew :app:assembleRelease -IS_SIGN_RELEASE_BUILD_WITH_DEBUG_KEY=true`

Reply:
@ccjernigan Thanks, that's a good idea. I've just changed the implementation to support this new property `IS_SIGN_RELEASE_BUILD_WITH_DEBUG_KEY`, see last commit. My question is if we are able to update also the related run configuration `ui-benchmark-test:connectedZcashmainnetBenchmarkAndroidTest` to automatically change the value of the new property to true?

Reply:
As far as I know, we can only set the property to true if we make that a Gradle run configuration instead of an Android run configuration.

Reply:
I thought so :/ So I propose to add this requirement to our benchmarking documentation of running it via the run configuration, at least.

Reply:
Agreed